alea/models/blueice_extended_model.py

```diff
@@ -21,27 +21,31 @@ class BlueiceExtendedModel(StatisticalModel):
     This class extends the `StatisticalModel` class and provides methods
     for generating data and computing likelihoods based on blueice.
 
+    Attributes:
+        parameters (Parameters): Parameters object containing the parameters of the model.
+        data (dict): Data of the statistical model.
+        is_data_set (bool): Whether data is set.
+        _likelihood (LogLikelihoodSum): A blueice LogLikelihoodSum instance.
+        likelihood_names (list): List of likelihood names.
+        data_generators (list): List of data generators for each likelihood term.
+
     Args:
         parameter_definition (dict): A dictionary defining the model parameters.
         likelihood_config (dict): A dictionary defining the likelihood.
+
+    Todo:
+        analysis_space could be inferred from the data
+        (assert that all sources have the same analysis space)
     """
 
     def __init__(self, parameter_definition: dict, likelihood_config: dict):
-        """Initializes the statistical model.
-
-        Args:
-            parameter_definition (dict): A dictionary defining the model parameters.
-            likelihood_config (dict): A dictionary defining the likelihood.
-        """
+        """Initializes the statistical model."""
         super().__init__(parameter_definition=parameter_definition)
         self._likelihood = self._build_ll_from_config(likelihood_config)
         self.likelihood_names = [t["name"] for t in likelihood_config["likelihood_terms"]]
         self.likelihood_names.append("ancillary_likelihood")
         self.data_generators = self._build_data_generators()
 
-    # TODO analysis_space could be inferred from the data
-    # (assert that all sources have the same analysis space)
```

Comment:
Ah I see that you removed these here.. I really like TODOs in the code since you can put it exactly where it belongs. Do you have strong opinions about not doing this?

Reply:
To use the rendered formatted Todo in docs.